tools/wpt/update_chromium_revision.py

```diff
@@ -0,0 +1,88 @@
+import requests
+from time import time
+
+from .utils import get
+
+
+PLATFORM_INFO = [
+    ("Win_x64", "chrome-win.zip"),
+    ("Win", "chrome-win.zip"),
+    ("Linux_x64", "chrome-linux.zip"),
+    ("Mac", "chrome-mac.zip")
+]
+SNAPSHOTS_PATH = "https://storage.googleapis.com/chromium-browser-snapshots/"
+
+
+def main(timeout=600.0):
+    start = time()
+
+    # Load existing pinned revision.
+    existing_revision = None
+    with open("tools/wpt/latest_chromium_revision.txt", "r") as f:
+        existing_revision = int(f.read())
+
+    # Find the lowest new revision number among latest revisions by platform.
```

Comment:
This isn't wrong, but it's a bit of an unnecessary optimization. How about just starting with the Linux revision and iterating from there? More requests might be needed to find a match, but it's simpler.

Reply:
> This isn't wrong

Taking a look again at the logic, I actually _did_ implement it wrong, as it's taking the `max` of the revisions instead of the `min`. 😅 

I've changed this to be implemented as you said - starting with Linux and just going there for simplicity's sake.